EESSI-remove-software.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# Script to remove part of the EESSI software stack (version set through init/eessi_defaults)
+
+# see example parsing of command line arguments at
+#   https://wiki.bash-hackers.org/scripting/posparams#using_a_while_loop
+#   https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
+
+display_help() {
+  echo "usage: $0 [OPTIONS]"
+  echo "  --build-logs-dir       -  location to copy EasyBuild logs to for failed builds"
```

Comment:
Does not seem to be used?

Reply:
Removed in https://github.com/EESSI/software-layer/pull/488/commits/6ca6a70671a019db0e83f54cb74e6049cba2a0f9.